tests/foreman/api/test_contentmanagement.py

```diff
@@ -73,6 +74,7 @@ class ContentManagementTestCase(APITestCase):
 
     @tier2
     @pytest.mark.skip("Uses old large_errata repo from repos.fedorapeople")
+    @skip_if(not settings.repos_hosting_url)
```

Comment:
Since you already added this it's not fair to ask to remove it now, but is that really necessary?

Personally, I'm not fan of this test skipping logic. On CI, these settings are always there. Locally, you either maintain these tests and have that config, or you don't and don't run them as well. If you need to run multiple tests outside of your area of expertise, you are better off copying config from CI to begin with.
I just struggle to find situations where these lines are useful.

Reply:
I am kind of agree with you about its use / not use. But QE's generally forgot to keep their properties file updated with the CI properties file. 

Even when this PR will merge, most of the QE's doing failure analysis will have many skips since they have not added this setting in their property. And if they don't add it, they will end up with unnecessary debugging the test failures if the validation of their tests isn't that strong to catch the issue quickly. So I am happy that I could save their time this way. :)

Also, due to any reason during switching the repo URL if they left it blank, they might again end up debugging the issue.

Reply:
I think its important to consider the behavior for anyone trying to run these test cases without the settings value present.  The config will just be loaded with a value of `None`, resulting in URL construction like `'None/fake_yum0'`. This will result in errors for the tests, of varying types depending on what the test is actually doing with that constant reference. 

I'd agree with @jyejare here that this will produce confusing results that will need to be evaluated - and that its a better option to skip the tests here.

What I would pretty much consider required here though, is to provide a reason when using `skipif`. That way the skip result has context, not just 'random' skips.

Reply:
@mshriver Make Sense, I would consider the skip in the next PR, and as you suggested using `@pytest.mark.requires_repo_host`  with the reason. That would again be a big refactor wrt lines.